### DISCUSSION TITLE
编译过程报错：没有可用的zlib，要怎么解决？

### DISCUSSION THREAD

**Discussion by dapeng111** [apache/doris#6596] [A - General / Q&A]
checking for library containing zlibVersion... no
configure: error: in `/root/incubator-doris/thirdparty/src/protobuf-3.5.1':
configure: error: --with-zlib was given, but no working zlib library was found

**Comment by morningman** [comment]
I suggest you to user docker dev image to build from Doris source.
It is more easy.
http://doris.incubator.apache.org/master/zh-CN/installing/compilation.html
